test/controllers/incomeSources/add/AddBusinessNameControllerSpec.scala

```diff
@@ -442,26 +455,26 @@ class AddBusinessNameControllerSpec extends TestSupport
       "the agent is authenticated and the business name entered is valid" in {
         disableAllSwitches()
         enable(IncomeSources)
-        setupMockGetSession(Some("value"))
+        setupMockCreateSession(true)
+        setupMockGetSessionKeyMongoTyped[String](Right(Some("testBusinessName")))
         val validBusinessName: String = "Test Business"
         setupMockAgentAuthRetrievalSuccess(testAgentAuthRetrievalSuccess, withClientPredicate = false)
         setupMockGetIncomeSourceDetails()(businessesAndPropertyIncome)
         val redirectUrl = controllers.incomeSources.add.routes.CheckBusinessDetailsController.showAgent().url
-        val redirect = Redirect(redirectUrl)
-        setupMockSetSession(SessionKeys.businessName, validBusinessName, redirect)
+        setupMockSetSessionKeyMongo(Right(true))
```

Comment:
verify if the right value was set with -> Mockito verify

Reply:
I was unable to get the verify working as it wouldn't match the call, accessed it directly through the mockSessionService instead